war/src/main/java/io/lumeer/engine/controller/CollectionMetadataFacade.java

```diff
@@ -354,36 +345,19 @@ public void addOrIncrementAttribute(String collectionName, String attributeName)
       Attribute attribute = getAttributeInfo(collectionName, attributeName);
 
       if (attribute != null) {
-         dataStorage.incrementAttributeValueBy(
-               metadataCollection(),
-               dialect.documentIdFilter(id),
-               dialect.concatFields(
-                     LumeerConst.Collection.ATTRIBUTES_KEY,
-                     attributePath(attributeName),
-                     LumeerConst.Collection.ATTRIBUTE_COUNT_KEY),
-               1);
+         throw new UnsupportedOperationException();
+         // TODO: use query
```

Comment:
Let's suppose `attributeName` is the full name here and update it accordingly. Or is there some problem I am not aware of?